wowchemy/assets/js/wowchemy-theming.js

```diff
@@ -180,7 +180,7 @@ function renderThemeVariation(isDarkTheme, themeMode = 2, init = false) {
       console.debug('Initializing Mermaid with light theme');
       if (init) {
         /** @namespace window.mermaid **/
-        window.mermaid.initialize({theme: 'default', securityLevel: 'loose'});
```

Comment:
Todo: add security level back

Reply:
Please explain why you have changed the Mermaid security level from "loose" to "strict"?

Reply:
@malmarz ?